code/core/src/components/components/tooltip/ListItem.tsx

```diff
@@ -200,11 +195,11 @@ const ListItem = ({
   LinkWrapper = undefined,
   ...rest
 }: ListItemProps) => {
-  const itemProps = getItemProps(onClick, href, LinkWrapper);
   const commonProps = { active, disabled };
+  const itemProps = getItemProps(onClick, href, LinkWrapper);
 
   return (
-    <Item {...commonProps} {...rest} {...itemProps}>
+    <Item {...rest} {...commonProps} {...itemProps}>
```

Comment:
is this a breaking change given that commonProps will always override rest?

Reply:
`commonProps` only contains `active` and `disabled` (see line 198), both of which are destructured off props so neither will ever exist on `rest`. So no this is not a breaking change.